iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -352,6 +352,19 @@ static SmallVector<int64_t> getDefaultDistributedLevelTileSizes(
   }
   return distributedLevelTileSizes;
 }
+static SmallVector<int64_t> getDefaultDistributedLevelTileSizes(
+    linalg::LinalgOp linalgOp, ArrayRef<int64_t> minTileSizes,
+    ArrayRef<int64_t> maxTileSizes, ArrayRef<int64_t> vectorSizeHints = {}) {
+  OpBuilder builder(linalgOp.getContext());
+  builder.setInsertionPoint(linalgOp);
+  SmallVector<Range> iterationDomain =
+      linalgOp.createLoopRanges(builder, linalgOp.getLoc());
```

Comment:
It seems to be unnecessary to create constant ops for loop ranges, since we only care about their static values. But looks like the upstream doesn't provide the tool to avoid that currently.

Reply:
We do have a method to return such stuff, it's called `getStaticLoopRanges`. The method infers range bounds for each dim.

Reply:
Cool! Can we use it here? For me, it is surprising that a method called "get*" will modify the IR.